client/main.lua

```diff
@@ -324,7 +324,27 @@ CreateThread(function()
 
 		if ClosestDoor and ClosestDoor.distance < ClosestDoor.maxDistance then
 			if Config.DrawTextUI and not ClosestDoor.hideUi and ClosestDoor.state ~= showUI then
-				lib.showTextUI(ClosestDoor.state == 0 and lockDoor or unlockDoor)
+				if ClosestDoor.state == 0 then
+					lib.showTextUI("[E] - Lock Door", {
```

Comment:
use lockDoor and unlockDoor rather than string literals.

Reply:
Done.